src/polymer/js-utils.ts

```diff
@@ -51,24 +52,54 @@ export function toScannedProperty(
     description: description,
     sourceRange: sourceRange,
     astNode: node, warnings,
-    private: !!name && (name.startsWith('_') || name.endsWith('_')),
+    isConfiguration: configurationProperties.has(name || ''),
+    jsdoc: parsedJsdoc
   };
 
-  if (node.kind === 'get' || node.kind === 'set') {
-    result.type = '';
+  const explicitPrivacy = jsdoc.getPrivacy(parsedJsdoc);
+  if (explicitPrivacy) {
+    result.privacy = explicitPrivacy;
+  } else if (result.name.startsWith('__')) {
+    result.privacy = 'private';
+  } else if (result.name.startsWith('_')) {
+    result.privacy = 'protected';
+  } else if (result.name.endsWith('_')) {
+    result.privacy = 'private';
+  } else {
+    result.privacy = 'public';
   }
 
   return result;
 };
 
+/** Properties on element prototypes that are purely configuration. */
```

Comment:
Can you expand the comment a little? At first read is hard to tell when this set would be used for.

Also, looking at what this is actually used for, rather than have an isConfiguration bit, I might just say that these are implicitly protected, which will hide them in the docs by default.